packages/components/src/Tooltip/Tooltip.mdx

```diff
@@ -34,16 +34,31 @@ import { Tooltip } from "@jobber/components/Tooltip";
 
 ## Design & Usage Guidelines
 
-When contributing to, or consuming the Tooltip component, consider the following:
+When contributing to, or consuming the Tooltip component, consider the
+following:
 
 ### Usage
-- A tooltip should be used to introduce a novel UI element or highlight a change to an existing interface.
-- A tooltip should contain information that is helpful, but not necessary for the user to successfully interact with the page.
+
+- A tooltip should be used to introduce a novel UI element or highlight a change
+  to an existing interface.
+- A tooltip should contain information that is helpful, but not necessary for
+  the user to successfully interact with the page.
 
 ### Content
+
 - Tooltip content should consist of only text and an optional "dismiss" CTA
 - Tooltip text content should not exceed 3 lines
 
+## Deficiencies
+
+### Accessibility
```

Comment:
I like this, going to close my PR

Reply:
This is maybe being pedantic but should "Accessibility" just be a level 2 heading? Maybe @GetJobber/frends could add it to the docs generator template 🤔

Reply:
My thought was that the "Deficiencies" section would be level 2 and there'd be "Accessibility" and "Technical" sections (maybe others?). Accessibility could be a level 2 heading if it described how to use accessibility features of the component.

Reply:
Ooh yeah that makes sense